packages/api/cms-api/src/generator/crud-generator.decorator.ts

```diff
@@ -41,19 +42,30 @@ export interface CrudFieldOptions {
 
 export function CrudField({
     resolveField = true,
+    rootArg = false,
     search = true,
     filter = true,
     sort = true,
     input = true,
 }: CrudFieldOptions = {}): PropertyDecorator {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     return function (target: any, propertyKey: string | symbol) {
-        Reflect.defineMetadata(`data:crudField`, { resolveField, search, filter, sort, input }, target.constructor, propertyKey);
+        Reflect.defineMetadata(`data:crudField`, { resolveField, rootArg, search, filter, sort, input }, target.constructor, propertyKey);
     };
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function hasFieldFeature(metadataClass: any, propName: string, option: keyof CrudFieldOptions): boolean {
     const crudField = (Reflect.getMetadata(`data:crudField`, metadataClass, propName) ?? {}) as CrudFieldOptions;
-    return crudField[option] ?? true;
+    const optionValue = crudField[option];
+    if (optionValue !== undefined) {
+        return optionValue;
+    } else {
+        //default value
+        if (option == "rootArg") {
+            return false;
+        } else {
+            return true;
+        }
+    }
```

Comment:
Why do you need this? I thought `??` only applies if `null` or `undefined` anyway?

Reply:
because rootArg should have false as default value, while the other options should have true.

(yes, that is confusing, but reversing the name to notRootArg is even more)